kernel/x86_64/dgemm_kernel_8x8_skylakex.c

```diff
@@ -429,7 +429,8 @@ static void KERNEL_MAIN(double *packed_a, double *packed_b, BLASLONG m, BLASLONG
     double *c_pointer = c;
     __mmask16 k01 = 0x00f0,k02 = 0x000f,k03 = 0x0033;
     BLASLONG ndiv8_count;
-    double *b_scratch = (double *)aligned_alloc(64,192*k);
+    double *b_scratch;
+    posix_memalign(&b_scratch,64,192*k);
```

Comment:
rather than replacing this, turning it into
```
#if defined(OS_WINDOWS)
 ... _aligned_malloc... (note the added undescore and "M") 
#else
... posix_memalign ...
#endif
```
would probably take care of platform issues ?

Reply:
Thank you very much for the suggestions. 
Later I found caching part of blocked B matrix into b_scratch actually slow down a bit... so no need for aligned allocation.